packages/teleport-publisher-vercel/src/crypto-node.js

```diff
@@ -0,0 +1,8 @@
+
+const { createHash } = require('crypto')
+
+export const encryptFileData = (fileData) => {
+    const hash = createHash('SHA1')
+    hash.update(fileData)
```

Comment:
I think this can be a one liner. 
`return createHash('SHA1').update(fileData).digest()`